devops/docker-compose.local.yml

```diff
@@ -88,27 +90,20 @@ services:
       - FLASK_APP=/app/geoapi/app.py
       - APP_ENV=local
       - ASSETS_BASE_DIR=/assets
-    stdin_open: true
-    tty: true
-    container_name: geoapicelerybeat
-    hostname: geoapicelerybeat
-    command: "celery -A geoapi.celery_app beat -l info"
+    container_name: geoapi_celerybeat
+    hostname: geoapi_celerybeat
+    command: "celery -A geoapi.celery_app beat -l info --uid=nobody --gid=nogroup"
     depends_on:
       rabbitmq:
         condition: service_healthy
 
-  api:
+  backend:
```

Comment:
README.md

`docker exec -it geoapi python initdb.py` to `docker exec -it geoapi-backend-1 python initdb.py`